src/elements/paginator/paginator/paginator.scss

```diff
@@ -132,7 +132,7 @@
     }
   }
 
-  &[data-selected] {
+  &:state(selected) {
```

Comment:
changes in this file need to be reverted as the buttons are in shadow DOM and still having data attributes (should fix the failing visual tests)